openlcs/packages/views.py

```diff
@@ -299,8 +299,11 @@ def import_package(self, request, *args, **kwargs):
                             status=status.HTTP_400_BAD_REQUEST)
         if serializer.is_valid():
             parent_task_id = request.data.get('parent_task_id', '')
+            token = request.data.get('token')
+            token_sk = request.data.get('token_sk')
```

Comment:
I'm a little bit concerned about this. not sure it's true: If both encrypted token stsring and the secret key were sent over network, it is equivalent to unencrypted.

Reply:
I'm not entirely sure yet if it's really an issue, to be honest.

Reply:
In non-local environment, we use https protocol.I think this is not an issue for now.I will merge it.If we found some security problem, i will fix it.